controllers/topic.controllers.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs/promises');
+const { selectTopics } = require('../models/topic.models');
+
+exports.getTopics = (req, res, next) => {
+	selectTopics()
+		.then((topics) => {
+			res.status(200).send({ topic: topics });
```

Comment:
The topics should be send back on a key of 'topics' but currently they are on a key of 'topic'.